### PR TITLE
Joel's branch

### PR DESCRIPTION
**Pull request by joelphilippage**
Fixed a few crashes and made small modifications.

### CODE REVIEW

File: Overheads/App.xaml.cs [hunk 1]
```diff
@@ -1,21 +1,49 @@
...
+    /// </summary>
+    public partial class App : Application
+    {
+        public bool DoHandle { get; set; }
+        private void Application_DispatcherUnhandledException(object sender, System.Windows.Threading.DispatcherUnhandledExceptionEventArgs e)
+        {
+            if (this.DoHandle)
+            {
+                //Handling the exception within the UnhandledExcpeiton handler.
+                MessageBox.Show(e.Exception.Message, "Exception Caught", MessageBoxButton.OK, MessageBoxImage.Error);
```
Comment (thepaulpage) [inline Overheads/App.xaml.cs]: sometimes it's helpful here to see the error message:
  "Exception Caught: " + e.Message

File: Overheads/Settings.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿namespace Overheads.Properties {
```
Comment (thepaulpage) [inline Overheads/Settings.cs]: there's no code here, is this required?

**Comment by JohnBloom** [comment]
Strange, all of the files look like they have changed. Im not sure why. It is hard to see what the actual changes are. I am fine with merging it in but I was curious about what changes were actually made.

**Comment by thepaulpage** [comment]
@JohnBloom we've been working on trying to fix that this afternoon, but aren't getting anywhere.  I linked him to hanselman's pink wall post and some other things, but...

**Comment by JohnBloom** [comment]
In that case just merge it. I dont think it is worth spending a ton of time on if he is actually using the software.

**Comment by JohnBloom** [comment]
@joelphilippage I could even add you to the repo as a contributor unless you wanted the MR code review.

**Comment by joelphilippage** [comment]
@JohnBloom Sure. Go ahead.

**Pull request merged**